app/views/admin/winners/index.blade.php

```diff
@@ -23,21 +23,18 @@
           <table class="table table-striped">
             <thead>
               <tr>
-                <th> ID </th>
-                <th> Name </th>
+                <th> ID (edit user info) </th>
```

Comment:
Oh yeah, since they are both links I added in that to make sure you knew what you were clicking on.